Header header
# Requested data
float32[] initial_state
float32[] in_predicted_state
float32[] in_predicted_input
float32[] online_data
# Received data
int32 status
float32 solution_time
float32 kkt_value
float32 objective_value
float32[] out_predicted_state
float32[] out_predicted_input